jaxley/modules/base.py

```diff
@@ -1242,12 +1251,12 @@ def add_to_group(self, group_name: str):
         Args:
             group_name: The name of the group.
         """
-        if group_name not in self.base.groups:
-            self.base.groups[group_name] = self._nodes_in_view
+        if group_name not in self.base.group_names:
```

Comment:
An issue with tracking groups in the nodes dataframe that could arise is that a group name conflicts with  other channel / param names. Could we add a check that ensure groups can only be added if a column of that name does not exist already?